src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -692,4 +693,28 @@ private function addCacheSection(ArrayNodeDefinition $rootNode)
             ->end()
         ;
     }
+
+    private function addPhpErrorsSection(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('php_errors')
+                    ->info('PHP errors handling configuration')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->booleanNode('log')
+                            ->info('Use the app logger instead of the PHP logger for logging PHP errors.')
+                            ->defaultValue($this->debug)
+                            ->treatNullLike($this->debug)
```

Comment:
You reverted the deprecation, but the value is still at debug, which looks very wrong to me still.